### ISSUE TITLE
[APP SUBMITTED]: HTTPConnectionPool(host='api.thesubdb.com', port=80): Read timed out. (read timeout=10)

### ISSUE THREAD

**Issue by duramato** [pymedusa/SickRage#306]
### INFO
Python Version: **2.7.6 (default, Jun 22 2015, 18:00:18) [GCC 4.8.2]**
Operating System: **Linux-3.19.0-56-generic-i686-with-Ubuntu-14.04-trusty**
Locale: UTF-8
Branch: **develop**
Commit: PyMedusa/SickRage@89a459923cfb2020bc32aab80527bc8a263731d5
No Log available with ERRORS:
### ERROR
```
17:54:41 ERROR::Thread-13 :: [89a4599] Provider 'thesubdb' timed out
Traceback (most recent call last):
  File "/home/home/SickRage/lib/subliminal/core.py", line 118, in list_subtitles_provider
    return self[provider].list_subtitles(video, provider_languages)
  File "/home/home/SickRage/lib/subliminal/providers/thesubdb.py", line 76, in list_subtitles
    return [s for s in self.query(video.hashes['thesubdb']) if s.language in languages]
  File "/home/home/SickRage/lib/subliminal/providers/thesubdb.py", line 56, in query
    r = self.session.get(self.server_url, params=params, timeout=10)
  File "/home/home/SickRage/lib/requests/sessions.py", line 480, in get
    return self.request('GET', url, **kwargs)
  File "/home/home/SickRage/lib/requests/sessions.py", line 468, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/home/SickRage/lib/requests/sessions.py", line 578, in send
    r = adapter.send(request, **kwargs)
  File "/home/home/SickRage/lib/requests/adapters.py", line 449, in send
    raise ReadTimeout(e, request=request)
ReadTimeout: HTTPConnectionPool(host='api.thesubdb.com', port=80): Read timed out. (read timeout=10)
```
---
_STAFF NOTIFIED_: @pymedusa/support @pymedusa/moderators

**Comment by duramato** [comment]
### INFO
Python Version: **2.7.6 (default, Jun 22 2015, 18:00:18) [GCC 4.8.2]**
Operating System: **Linux-3.19.0-56-generic-i686-with-Ubuntu-14.04-trusty**
Locale: UTF-8
Branch: **develop**
Commit: PyMedusa/SickRage@89a459923cfb2020bc32aab80527bc8a263731d5
No Log available with ERRORS:
### ERROR
```
17:54:06 ERROR::Thread-13 :: [89a4599] Unexpected error in provider 'addic7ed'
Traceback (most recent call last):
  File "/home/home/SickRage/lib/subliminal/core.py", line 118, in list_subtitles_provider
    return self[provider].list_subtitles(video, provider_languages)
  File "/home/home/SickRage/lib/subliminal/providers/addic7ed.py", line 269, in list_subtitles
    return [s for s in self.query(video.series, video.season, video.year)
  File "/home/home/SickRage/lib/subliminal/providers/addic7ed.py", line 234, in query
    raise TooManyRequests()
TooManyRequests
```
---
_STAFF NOTIFIED_: @pymedusa/support @pymedusa/moderators

**Comment by ratoaq2** [comment]
These 2 errors are reasonable and it should be logged as error.
My suggestion is: if you want to ignore these errors, better to turn off subliminal logs in the 
`configuration -> advanced` after this merge #310 

What do you guys think about this? Any other suggestion?

**Comment by duramato** [comment]
I dont agree with them beeing an error, all time outs and simliars should be warnings

**Comment by ratoaq2** [comment]
This was fixed by #310 
For providers `addic7ed`, `itasa` and `tvsubtitles`: errors -> info, warning -> debug
For other providers: errors -> warning, warning -> info
